src/modal.test.basics.ts

```diff
@@ -174,3 +190,33 @@ it('has `severity="medium" coverage', async () => {
     </glide-core-modal>`,
   );
 });
+
+it('has #onScroll() coverage', async () => {
```

Comment:
Was hoping for a simpler:

```js
it('has #onScroll() coverage', async () => {
  const host = await fixture<Modal>(
    html`<glide-core-modal label="Label" open>
      <div
        style="height: 100vh; display: flex; flex-direction: column; justify-content: flex-end;"
      >
        <button>button</button>
      </div>
    </glide-core-modal>`,
  );

  host?.querySelector('button')?.scrollIntoView({ behavior: 'instant' });

  // Wait for the scroll to finish.
  await aTimeout(0);
});
```

But it would flake every 1/4 runs. Going this route, while more complex, ensures it consistently passes.